xbmc/addons/kodi-addon-dev-kit/tools/deploy-documentation.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+cd $TRAVIS_BUILD_DIR/xbmc/addons/kodi-addon-dev-kit/doxygen
+
+echo "Generating addon-dev-kit doxygen documentation ..."
+
+doxygen
+if [ $? != 0 ]; then
+  exit 1
+fi
+
+echo "Starting upload of doxygen documentation"
+cd $TRAVIS_BUILD_DIR/xbmc/addons/kodi-addon-dev-kit/docs/html
+htmlFiles=$(find | sed 's/\.\///')
+htmlDirs=$(find -mindepth 1  -type d | sed 's/\.\///')
+if [ ${#htmlFiles[@]} -eq 0 ]; then
+  echo "No files to update"
+else
+  lftp ftp://$FTP_USER:$FTP_PASS@$DOC_UPLOAD_URL -e "mirror -e -R . /$DOC_UPLOAD_PATH ; quit"
```

Comment:
This do the ftp upload for a website, but not like ftp ;)
Maybe for Kodi's side it can be changed to another upload way.